source/tyr/tyr/resources.py

```diff
@@ -366,6 +367,8 @@ def put(self, id=None, name=None):
                             location=('json', 'values'), default=instance.night_bus_filter_base_factor)
         parser.add_argument('priority', type=int, help='instance priority',
                             location=('json', 'values'), default=instance.priority)
+        parser.add_argument('bss_provider', type=bool, help='bss provider activation',
+                            location=('json', 'values'), default=True)
```

Comment:
We prefer to use the previous value as default when updating, it's allow us to do partial update